### PR TITLE
bugfix: chat feature

### PR DESCRIPTION
**Pull request by khairul-alam-licon**
# Description
This PR aims to fix some issues according to the following Links.

## Links
- MINI-3087
- MINI-3088
- MINI-3092
- MINI-3096
- MINI-3099

# Checklist
- [x] I have read the [contributing guidelines](../.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](../.github/CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

**Review by corycaywood — APPROVED** [review]
LGTM 👍

File: testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/chat/ChatWindow.kt [hunk 1]
```diff
@@ -81,33 +78,34 @@ class ChatWindow(private val activity: Activity) {
...
     private fun prepareWindow(mode: ContactSelectionMode) {
         val rootView = WindowChatBinding.inflate(layoutInflater, null, false)
+        contactSelectionAlertDialog =
+            AlertDialog.Builder(activity, R.style.AppTheme_DefaultWindow).create()
+        contactSelectionAlertDialog.setView(rootView.root)
 
-        // set message content
-        GlobalScope.launch(Dispatchers.Main) {
-            message.apply {
-                rootView.messageImage.load(activity, image, R.drawable.r_logo)
```
Comment (khairul-alam-licon) [inline testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/chat/ChatWindow.kt:102]: Looks like, Glide has some issues with thread while loading image from different kind of urls. Removing this line temporarily fix the crash (MINI-3099), but currently it's showing the default icon. I am still working this.

**Review by minh-rakuten — APPROVED** [review]
👍 👍

**Comment by khairul-alam-licon** [comment]
Thank you for review guys 👍

**Pull request merged**